virtio-queue/src/queue/verification.rs

```diff
@@ -0,0 +1,270 @@
+use std::mem::ManuallyDrop;
```

Comment:
Minor nitpick:

```suggestion
// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
use std::mem::ManuallyDrop;
```

Let's keep all files consistent

Otherwise LGTM.

Reply:
Over at firecracker we have a home-grown python script that checks the license headers: https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/style/test_licenses.py. Might be adapable?

Reply:
Added license. Resolving.

Reply:
> Over at firecracker we have a home-grown python script that checks the license headers: https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/style/test_licenses.py. Might be adapable?

mmm, yeah, it could be a good starting point